pkg/integration/deprecated/integration.go

```diff
@@ -560,5 +560,7 @@ func getLazygitCommand(testPath string, rootDir string, mode Mode, speed float64
 		cmdObj.AddEnvVars(fmt.Sprintf("REPLAY_EVENTS_FROM=%s", replayPath))
 	}
 
+	cmdObj.AddEnvVars("GIT_CONFIG_GLOBAL=/dev/null")
+
```

Comment:
I thought it was a good idea to do this for the legacy tests as well, but it turns out that it isn't, since the global config [is changed on CI](https://github.com/jesseduffield/lazygit/blob/master/.github/workflows/ci.yml#L78) for these. So taking this out again.